app/views/index.erb

```diff
@@ -4,7 +4,7 @@
     <div class="span-8 hero-unit" style="margin-top: 3em;">
       <h1>Hej.</h1>
       <p class="pull-left" style="margin-right: 1em;">
-        Have a look at the places <%= format_travellers @travellers %> have been to.
+        Have a look at the places <%= format_travellers(@travellers).html_safe %> have been to.
```

Comment:
Bonus points for spotting that. :blush: 
(Also: why is there no :broom: Emoji?)